README.md

```diff
@@ -1,86 +1,80 @@
 conda activate parastell_env
 conda install -c conda-forge mamba
 ```
 
 The subsequent mamba and pip install commands should be run with this environment activated.
 
-### Install Dependencies using Mamba and Pip
-Install ParaStell dependencies available on conda-forge.
+Mamba install ParaStell and PyStell-UW Python dependencies available on `conda-forge`:
 
 ```
-mamba install -c conda-forge cadquery moab
+mamba install -c conda-forge numpy scipy scikit-learn cadquery cad_to_dagmc matplotlib
+mamba install -c conda-forge moab=5.5.0=nompi_tempest_*
 ```
 
-Install pystell_uw dependencies.
-
-```
-mamba install matplotlib
-```
-
-Pip install remaining pystell_uw dependency.
+Pip install the remaining ParaStell and PyStell-UW Python dependencies:
 
 ```
 pip install netCDF4
+pip install pyyaml
 ```
 
-Pip install remaining ParaStell dependencies.
-
-```
-pip install cad-to-dagmc
-pip install gmsh
-```
-
-### Coreform Cubit
-Download and install version 2023.11 from [Coreform's Website](https://coreform.com/products/downloads/), then add the /Coreform-Cubit-2023.11/bin/ directory to the `PYTHONPATH` by adding a line to the .bashrc file like the following:
+### Install Coreform Cubit
+Download and install version 2023.11 from [Coreform's Website](https://coreform.com/products/downloads/), then add the `/Coreform-Cubit-2023.11/bin/` directory to your `PYTHONPATH` by adding a line similar to the following to your `.bashrc` file:
 
-```
+```bash
 export PYTHONPATH=$PYTHONPATH:$HOME/Coreform-Cubit-2023.11/bin/
 ```
 
-Replace $HOME with the path to the Cubit directory on your system. Additional information about adding modules to the `PYTHONPATH` can be found [here](https://www.tutorialspoint.com/How-to-set-python-environment-variable-PYTHONPATH-on-Linux).
+Replace `$HOME` with the path to the Coreform Cubit directory on your system. Additional information about adding modules to your `PYTHONPATH` can be found [here](https://www.tutorialspoint.com/How-to-set-python-environment-variable-PYTHONPATH-on-Linux).
 While it is possible to use ParaStell with older versions of Cubit, additional steps not in this guide may be required.
 
-If you do not have a Cubit license, you may be able to get one through [Cubit Learn](https://coreform.com/products/coreform-cubit/free-meshing-software/) at no cost.
+If you do not have a Coreform Cubit license, you may be able to get one through [Cubit Learn](https://coreform.com/products/coreform-cubit/free-meshing-software/) at no cost.
 
-### VMEC Tools
-Download and extract the repository for pystell_uw using
+### Install PyStell-UW
+Download and extract the PyStell-UW repository:
 
-```
-git clone https://github.com/aaroncbader/pystell_uw.git
+```bash
+git clone --single-branch --branch old https://github.com/aaroncbader/pystell_uw.git
 ```
 
-or download the and extract the zip from [pystell_uw](https://github.com/aaroncbader/pystell_uw). Once extracted, add the directory to the `PYTHONPATH`.
```

Comment:
perhaps update this link to go to the old branch, and maybe add a note about why the old branch is necessary, though I would expect most folks will use git to clone rather than downloading the zip
https://github.com/aaroncbader/pystell_uw/tree/old